Python/Product/PythonTools/PythonTools/Project/ImportWizard/ImportSettings.cs

```diff
@@ -216,7 +218,7 @@ internal class ImportSettings : DependencyObject {
                 }
             }
 
-            StartupFile = SelectDefaultStartupFile(fileList, StartupFile);
```

Comment:
The difference between `this.StartupFile` above vs `StartupFile` here caught my eye. There's no actual difference, of course - but when reading the code, due to the distinction, one might assume that there is one, and it's important enough to highlight. Let's keep them consistent for clarity.

Reply:
@int19h Just updated `this.StartupFile` to `StartupFile` to be consistent.